app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CartController < ApplicationController
+  def index
+    @cart
+  end
```

Comment:
```suggestion
  def index; end
```

Tu n'as même pas besoin de retourner @cart ici vu qu'il est défini dans un before